docs/tuning-guide.md

```diff
@@ -294,3 +294,35 @@ performance.
 - [`spark.rapids.sql.hasNans`](configs.md#sql.hasNans)
 - [`spark.rapids.sql.castFloatToString.enabled`](configs.md#sql.castFloatToString.enabled)
 - [`spark.rapids.sql.castStringToFloat.enabled`](configs.md#sql.castStringToFloat.enabled)
+
+## Metrics
+
+Custom Spark SQL Metrics are available which can help identify performance bottlenecks in a query.
+
+| Key              | Name                     | Description                                       |
+|------------------|--------------------------|---------------------------------------------------|
+| bufferTime       | buffer time              | Time spent buffering input from file data sources. |
+| buildDataSize    | build side size          | Size in bytes of the build-side of a join.        |
+| buildTime        | build time               | Time to load the build-side of a join.            |
+| collectTime      | collect batch time       | Same as fetchTime                                 |
+| computeAggTime   | aggregation time         | Time performing aggregation.                      |
+| concatTime       | concat batch time        | Time to concatenate batches.                      |
+| fetchTime        | fetch time               | Time spent fetching data from child operator(s).  |
```

Comment:
This needs to be deleted after the latest changes on #2515

Reply:
This has now been deleted, and the definition for collectTime has been updated.